maps/nerva/nerva_holodecks.dm

```diff
@@ -80,4 +90,55 @@
 			"Wildlife Simulation" = "wildlifecarp"
 		)
 
-	)
\ No newline at end of file
+	)
+
+// Holo-Turf
+
+/turf/simulated/floor/holofloor/urist/junglegrass_lightgreen
+	name =  "wet grass"
+	desc = "Perfectly simulated hot and humid climate included!"
+	base_desc = "Perfectly simulated hot and humid climate included!."
+	icon = 'icons/jungle.dmi'
+	icon_state = "grass4"
+
+/turf/simulated/floor/holofloor/urist/junglegrass_green
+	name =  "wet grass"
+	desc = "Perfectly simulated hot and humid climate included!"
+	base_desc = "Perfectly simulated hot and humid climate included!."
+	icon = 'icons/jungle.dmi'
+	icon_state = "grass3"
+
+/turf/simulated/floor/holofloor/urist/junglegrass_darkgreen
+	name =  "wet grass"
+	desc = "Perfectly simulated hot and humid climate included!"
+	base_desc = "Perfectly simulated hot and humid climate included!."
+	icon = 'icons/jungle.dmi'
+	icon_state = "grass2"
+
+/turf/simulated/floor/holofloor/urist/junglegrass_browngreen
+	name =  "wet grass"
+	desc = "Perfectly simulated hot and humid climate included!"
+	base_desc = "Perfectly simulated hot and humid climate included!."
+	icon = 'icons/jungle.dmi'
+	icon_state = "grass1"
+
+/turf/simulated/floor/holofloor/urist/jungle_water
+	name =  "murky water"
+	desc = "Holographic deep, dank water. You swear you can see something moving below occasionally."
+	base_desc = "Holographic deep, dank water. You swear you can see something moving below occasionally."
+	icon = 'icons/urist/jungle/turfs.dmi'
+	icon_state = "rivernew"
+
+/turf/simulated/floor/holofloor/urist/pavement
+	name = "Pavement"
+	desc = "Holographic deep, dank water. You swear you can see something moving below occasionally."
```

Comment:
i'm not sure this is a proper description for pavement

Reply:
no more dank pavements

Reply:
Ooops, haha